packages/myst-frontmatter/src/frontmatter/validators.ts

```diff
@@ -389,9 +394,6 @@ export function validateAffiliation(input: any, opts: ValidationOptions) {
   if (defined(value.name)) {
     output.name = validateString(value.name, incrementOptions('name', opts));
   }
-  if (defined(value.institution)) {
-    output.institution = validateString(value.institution, incrementOptions('institution', opts));
-  }
   if (defined(value.department)) {
     output.department = validateString(value.department, incrementOptions('department', opts));
```

Comment:
does this have an impact of fields available to exports, e.g. web, latex templates...?